cmd/limactl/template.go

```diff
@@ -95,6 +96,25 @@ func fillDefaults(ctx context.Context, tmpl *limatmpl.Template) error {
 	return err
 }
 
+func newTemplateURLCommand() *cobra.Command {
```

Comment:
Needs docs and tests

Reply:
Docs can be added under https://deploy-preview-3972--lima-vm.netlify.app/docs/config/plugin after merging:
- #3972

Reply:
This PR is also modifying the same logic that #3973 is currently changing.

And tests should be implemented using BATS from #3969.

I'm turning this back unto "Draft" until these 3 PRs have landed because it will likely need serious refactoring by then.